numpy/fft/tests/test_pocketfft.py

```diff
@@ -42,6 +42,41 @@ def test_fft(self):
         assert_allclose(fft1(x) / 30.,
                         np.fft.fft(x, norm="forward"), atol=1e-6)
 
+        # tests below only test the out parameter
+        y = random((30, 20)) + 1j*random((30, 20))
+
+        out = np.zeros_like(x, dtype=complex)
+        assert_allclose(fft1(x), np.fft.fft(x, out=out), atol=1e-6)
```

Comment:
For all these tests, you need to check too that `out` is actually returned, i.e., have something like
```
result = np.fft.fft(x, out=out)
assert result is out
assert_array_equal(fft1(x), result)
```
I replaced also with `assert_array_equal` since, hopefuilly, fft code is reproducible on a given machine!